Dockerfile

```diff
@@ -1,17 +1,17 @@
-FROM gradle:jdk11 as cache
+FROM gradle:8.5-jdk21 as cache
```

Comment:
Why the use of `[8.5-jdk21](https://hub.docker.com/layers/library/gradle/8.5-jdk21/images/sha256-6fbc64efc26b6b092b7ba0b2584de23985817edc65e0db6f105e1bbfb83b5314)` (updated a year ago) over of `[jdk21](https://hub.docker.com/layers/library/gradle/jdk21/images/sha256-fa5b6583b7850a0745264d7905213d648afb7ad9ae07f4f2f8c2568be8eebf44)` (updated recently)?

Reply:
8.5-jdk21 is a much smaller image to download as it only contains the two images we need:

<img width="1462" alt="image" src="https://github.com/user-attachments/assets/056a1e71-4eaa-4517-9048-bf9268a07abc" />

where as `jdk21` contains additional images

<img width="1447" alt="image" src="https://github.com/user-attachments/assets/dd0b1653-399e-4bbd-a975-4b853b4866e3" />

Previous tests showed that there was considerable difference in the build times if the docker image specifying the gradle version was used.

I'll double check that this is still the case now and comment futher

Reply:
Thanks for looking into it and making the change.